extensions/wikia/Email/i18n/WallMessage.i18n.php

```diff
@@ -0,0 +1,54 @@
+<?php
+$messages = array();
+
+$messages['en'] = array(
+	'emailext-wallmessage-owned-subject' => '$1 wrote you a new message on {{SITENAME}}',
+	'emailext-wallmessage-following-subject' => '$1 wrote $2 a new message on {{SITENAME}}',
+	'emailext-wallmessage-owned-summary' => "'''$1 wrote you a new message on {{SITENAME}}. Lucky you!'''",
+	'emailext-wallmessage-following-summary' => "'''$1 wrote $2 a new message on {{SITENAME}}.'''",
+	'emailext-wallmessage-full-conversation' => 'Full conversation',
+	'emailext-wallmessage-recent-messages' => '[$1 All recent messages on {{SITENAME}}]',
+);
+
+$messages['qqq'] = array(
+	'emailext-wallmessage-owned-subject' => 'Email subject when another user wrote a new message on your wall. $1 is the name of the user who posted the message.',
+	'emailext-wallmessage-following-subject' => "Email subject when a user wrote a new message on another user's wall, and you are following the thread. $1 is the name of the user who posted the message, $2 is the wall owner (user).",
+	'emailext-wallmessage-owned-summary' => 'Text describing that another user wrote a new message on your wall. $1 is the name of the user who posted the message',
+	'emailext-wallmessage-following-summary' => "Text describing that a user wrote a new message on another user's wall. $1 is the name of the user who posted the message, $2 is the wall owner (user).",
+	'emailext-wallmessage-full-conversation' => 'Text for button that, when clicked, navigates to the full message wall conversation',
+	'emailext-wallmessage-recent-messages' => "Call to action to view all recent messages on the user's message wall",
+);
```

Comment:
I'm not sure if you added the following empty message arrays or if crowdin did, but in case you did, just note that crowdin will handle it for you ;)